css/style.css

```diff
@@ -15,5 +18,118 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+.navbar {
+  position: fixed;
+  right: 60px;
+  font-size: 10px;
+  width: 50%;
+}
+.navbar > ul {
+  display: flex;
+  justify-content: space-around;
+  padding: 20px;
```

Comment:
```suggestion
```

Reply:
I added the padding to avoid the misalignment of the logo and the navbar. 
![Screenshot 2020-12-12 at 22 23 55](https://user-images.githubusercontent.com/34775806/101996278-e222f680-3cc8-11eb-8028-71f9c97fb4e6.png)